configure.ac

```diff
@@ -1056,18 +1056,21 @@ then
     AC_MSG_ERROR([please disable dsa if disabling asn.])
 fi
 
-if test "$ENABLED_ECC" = "yes" && test "$ENABLED_ASN" = "no"
+if test "x$ENABLED_CRYPTONLY" = "xyes"
```

Comment:
Why does cryptonly influence whether asn and ecc are dependent?  Does PSK make sense in cryptonly?